Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.5-alpine
+FROM ruby:2.7.1-alpine
 
```

Comment:
@suz-stripe can you please chime in on what steps are needed to push a new Docker image to DockerHub?

Reply:
I can build and push manually for this PR merge, and will set up some magic plumbing for future merges so that it's set and forget.

LGTM 👍